clique/src/lib.rs

```diff
@@ -58,6 +58,9 @@ pub trait AddressingInformation: Debug + Hash + Codec + Clone + Eq + Send + Sync
 
     /// Verify the information.
     fn verify(&self) -> bool;
+
+    // Address used by the underlying protocol, for debugging purposes.
+    fn lower_level_address(&self) -> Option<String>;
```

Comment:
Either just call it `address` and have the comment say that it's _only_ used for debugging/logging (slightly more forcefully than now), or call it something that corresponds to the function better (I'm not sure what it would be, I only have extremely generic ideas like `additional_data`, which make it very clear that this does not impact behaviour, but are otherwise atrocious; just pointing out this option if you have some good idea).